src/go/rpk/pkg/cli/cmd/acl/create.go

```diff
@@ -65,19 +65,30 @@ Allow write permissions to user buzz to transactional id "txn":
 			}
 			types.Sort(results)
 
-			tw := out.NewTable(headersWithError...)
-			defer tw.Flush()
-			for _, c := range results {
-				tw.PrintStructFields(aclWithMessage{
-					c.Principal,
-					c.Host,
-					c.Type,
-					c.Name,
-					c.Pattern,
-					c.Operation,
-					c.Permission,
-					kafka.ErrMessage(c.Err),
-				})
+			createdACLS := createdACLCollection{}
```

Comment:
Based on this, and several other comments like it, it looks like all the "init to default value" lines should be updated to just be `var foo type` since that does the same thing. I'll try to make sure I get them all even if they weren't explicitly commented on.

Reply:
Thanks! also see the why here: https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

^ That explains the slices, but we try to be consistent for all types.